components/server/src/init.ts

```diff
@@ -108,8 +108,9 @@ export async function start(container: Container) {
 
     process.on("SIGTERM", async () => {
         log.info("SIGTERM received, stopping");
-        await server.stop();
         clearInterval(interval);
+        await server.stop();
+        process.exit(0);
```

Comment:
I'm not sure about these changes, since there can be important operations which should complete without leaving system in undefined state, i.e. requests in run to start workspace, rolling back some spice db changes, cascade deleting operations which don't have overall transaction, and so on. Stopping accepting new connections and letting event loop to run to the end seems to be desirable. 

Otherwise I think we can land it.

Reply:
Those are all problems that exist, but it doesn'T really matter if we kill everyone without notice 30 seconds later.

Reply:
It would be good, though if we can be sure that all participants on the stop signal await

Reply:
true given that in prod we still accept new connections